bokehjs/src/coffee/api/typings/models/scales.d.ts

```diff
@@ -0,0 +1,3 @@
+declare namespace Bokeh {
+
+}
```

Comment:
@mattpap need your help here. I think the `mappers.d.ts` file also needs updating not that color mapper inherits from `Transform`